tools/scripts/mysql_migration.py

```
@@ -1227,12 +1237,776 @@ def execute(self) -> tuple[int, list]:
+        lookup = {}
+        for model_id, model_name, model_type, tenant_id, provider_name in cursor.fetchall():
+            # model_type is a binary integer; we check each bit
+            for type_str, type_bit in self.MODEL_TYPE_TO_INT.items():
+                if model_type & type_bit:
+                    key = (tenant_id, model_name, provider_name, type_str)
+                    lookup[key] = model_id
+        return lookup
+
+    def _resolve_model_id(self, model_lookup: dict, tenant_id: str, model_name: str, model_type_str: str) -> str | None:
+        """Look up tenant_model.id from the model_lookup cache."""
+        pure_name, instance_name, provider_name = self._split_model_name(model_name)
```

Ruff (0.15.20)

[warning] Unpacked variable `instance_name` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)